packages/fxa-auth-server/lib/routes/subscriptions/mozilla.ts

```diff
@@ -34,7 +32,7 @@ export const mozillaSubscriptionRoutes = ({
   playSubscriptions?: PlaySubscriptions;
 }): ServerRoute[] => {
   if (!playSubscriptions) {
-    playSubscriptions = new PlaySubscriptions(config);
+    playSubscriptions = new PlaySubscriptions();
```

Comment:
Let's change to a `Container.get`.

Reply:
Ah yes since `Container.get` will instantiate the module if it isn't set in the container yet.